app/code/Magento/Catalog/Model/Product/Option/Validator/Select.php

```diff
@@ -16,7 +16,7 @@ class Select extends DefaultValidator
      * @param array $values
      * @return bool
      */
-    protected function checkAllValuesRemoved($values)
+    private function checkAllValuesRemoved($values)
```

Comment:
> this method is not called anywhere outside this class
But someone may extends from it and use this method.

Now inheritance is not recommended in Magento and delegation with Dependency Injection should be used. However while we have legacy code we should preserve backward compatibility. In future we will refactor such code.